app/javascript/components/pages/JobsAuthenticate.jsx

```diff
@@ -38,39 +39,49 @@ const JobsAuthenticate = () => {
         <SharedLayout>
             <div className="jobs-authenticate-container">
                 <Logo className="h-28" />
-                <form className="jobs-authenticate-login" onSubmit={(e) => handleLogin(e)}>
-                    Password
-                    <input
-                        type="password"
-                        className="rounded-lg mt-3"
-                        value={password}
-                        onChange={(e) => setPassword(e.target.value)}
-                    ></input>
-                    <Button type="secondary" className="w-40 mt-3">
-                        <input type="submit" value="View Job Board" className="bg-transparent" />
-                    </Button>
-                    {hasError && <div className="incorrect-password">Password is incorrect.</div>}
-                    <p className="text-sm mt-5">
-                        The WNB.rb job board is password protected. You can find the password in the
-                        WNB.rb Slack workspace. To join WNB.rb on Slack,{' '}
-                        <a
-                            href="https://tinyurl.com/join-wnb-rb"
-                            target="_blank"
-                            rel="noreferrer noopener"
-                            className="underline"
-                        >
-                            fill out this form.
-                        </a>
-                    </p>
-                </form>
-                <div className="jobs-authenticate-sponsor-us">
-                    Want to post a job?
-                    <Button type="white">
-                        <a href="/sponsor-us" target="_blank" rel="noopener noreferrer">
-                            Sponsor Us
-                        </a>
-                    </Button>
-                </div>
+                <Card className="w-full max-w-[30rem] mt-5">
```

Comment:
Almost everything in this component is the same, I just moved it into a `Card` for styles.